### DISCUSSION TITLE
Where do exported spritemaps save on mobile?

### DISCUSSION THREAD

**Discussion by pyroboots** [nesbox/TIC-80#2253] [Q&A]
Im trying to export a modified version of tic80's config spritemap so I can share it, but I can't find the exported image in my download, the android package folder, documents to name just a few.

**Comment by MateusH7** [comment]
Hi you May need to dowload another file manager at least in my Android it will not show on the default file system,if you download another  then you have to go on a folder -android- then inside -data- then you will have to find one called "com.nesbox.tic", this is for android I dont know how it works on ios.
